packages/fx-core/resource/package.nls.json

```diff
@@ -540,6 +540,9 @@
   "error.appstudio.publishInDevPortalSuggestionForValidationError": "Please try to generate package from \"Zip Teams app package\" and try again.",
   "error.appstudio.teamsAppCreateConflict": "Failed to create Teams app with 409 Conflict error. That may come from your app id is conflicting with another app in your tenant. Click Get Help to learn more.",
   "error.appstudio.teamsAppCreateConflictWithPublishedApp": "A Teams app with that id already exists in your organization's app store. Please manually update the app id and try again.",
+  "error.appstudio.NotAllowedToAcquireBotFrameworkToken": "The current account is not allowed to acquire botframework token.",
```

Comment:
Please help add comment for uncommon terms to make sure localization team is clear about whether each piece should be translated (like whether botframework should be translated here). For the commenting, you can reference **JSON/RESJSON File** part in aka.ms/commenting

Reply:
Thanks @HuihuiWu-Microsoft , I will check that later.